adapters/adform/adform.go

```diff
@@ -317,6 +352,12 @@ func openRtbToAdformRequest(request *openrtb.BidRequest) (*adformRequest, []erro
 			secure = true
 		}
 
+		_, valid := isPriceTypeValid(adformAdUnit.PriceType)
```

Comment:
You can remove this logic from the OpenRTB code path now. Those JSON schemas are enforced, so PBS will return a 400 before it even calls your Bidder if it's invalid.

You _do_ need to validate on the legacy path above, though.